openhtf/names.py

```diff
@@ -55,5 +56,6 @@
 # Classes used in writing test scripts.
 Measurement = openhtf.util.measurements.Measurement
 OutputToJSON = openhtf.io.output.json_factory.OutputToJSON
+UploadJSON = openhtf.io.output.upload_json.UploadJSON
```

Comment:
If there were a generic way to 'upload', then sure. But at the moment, your UploadJSON uploads in a specific manner, and all upload paths will be different. Be it over HTTP, raw TCP, etc and with OAuth, OAuth2, Basic Auth, etc. There are a ton of variations and trying to make a generic one will never be generic enough. Instead, we provide json output as human-readable only (not even guaranteed to be the same structure across versions, yet) and these other modules should only do the data-conversion necessary to convert the test_record data into their format and then upload.